.gitignore

```diff
@@ -64,3 +64,9 @@ package-lock.json
 # https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control
 #
 ios/Pods/
+
+#Generated files
+#
+generated/
+intermediates/
+outputs/
```

Comment:
Where do these directories actually appear -- from your PR message, it sounds like they'd be somewhere under `android/`, right?

We should make the patterns more specific than this so they refer only to the paths we actually mean. As is, these would cover any directory named e.g. "intermediates" anywhere in the tree, which could have surprising results.

Reply:
@gnprice No, Actually they appeared in my project's root directory after the build process. As a result I added these folders to .gitignore so as to avoid them being tracked by git. In short all the ignore configs added above are correct since they help me to ignore those auto generated files.

Reply:
@jainkuniya did you ever get these directories generated for you?

Reply:
Nope! 
For me they are created at `android/app/build` if I build APK.
<img width="204" alt="screen shot 2018-04-09 at 10 01 57 am" src="https://user-images.githubusercontent.com/18511177/38479539-85a62df8-3bdd-11e8-9598-75833a9b8275.png">
<img width="319" alt="screen shot 2018-04-09 at 10 05 14 am" src="https://user-images.githubusercontent.com/18511177/38479540-85e1c0b6-3bdd-11e8-8a92-c91d011e4fb0.png">

@HarshulSharma000 Can you try once deleting this folder and re-running?
And if they again got created, then what is it inside, are they empty?

Reply:
@jainkuniya @borisyankov  I did try again and this time no files were generated. But the first time they did. As I recall for the first time I had to do other config regarding the gradle version and project setup. But this time since it's all setup according the zulip project the same is not happening.